app/controllers/declaration/ItemsSummaryController.scala

```diff
@@ -88,11 +89,19 @@ class ItemsSummaryController @Inject()(
               Future.successful(BadRequest(itemsSummaryPage(mode, itemSummaryForm.fill(validYesNo), request.cacheModel.items.toList, incorrectItems)))
 
             case YesNoAnswers.no =>
-              Future.successful(navigator.continueTo(mode, controllers.declaration.routes.WarehouseIdentificationController.displayPage))
+              Future.successful(navigator.continueTo(mode, nextPage(request.declarationType)))
         }
       )
   }
 
+  private def nextPage(declarationType: DeclarationType): Mode => Call =
+    declarationType match {
```

Comment:
Checked out the branch and tested the navigation for one of each of these dec types.  All looks good